narps_open/utils/description/analysis_pipelines_full_descriptions.tsv

```diff
@@ -2301,7 +2301,7 @@ amygdala: as described above including the following maps: SF, MF, IF, LB, CM (l
+3. spatial smoothing "	NA	NA	NA	SPM12, Realign & Unwarp using the Phase map created with SPM12 Fieldmap Toolbox v2.1 (default options): Other than defaults: Estimation: Quality 0.95, Separation 3, Register to mean, Interpolation 7th Degree B-Spline; Unwarp Reslice: Interpolation 7th Degree B-Spline 		The single-band reference EPI was also distortion corrected using the SPM12 Fieldmap Toolbox v2.1 	NA	NA	SPM12: Within each run, the distortion corrected single-band reference EPI was co-registered to the mean EPI from Realignment & Warp using normalised mutual information. Then distortion corrected single-band reference EPI was co-registered to the gray matter probability map in the Old Segmentation toolbox in SPM using normalised mutual information and the distortion corrected EPI time-series as well as the mean EPI remained aligned. The single-band reference EPI was normalized to the SPM MNI152 template space using the classic Unified Segmentation approach in the Old Segment function in SPM, while mitigating overfitting by setting the warp frequency cutoff to 45 limiting the discrete cosine transform (DCT) bases and setting the sampling distance to 2. The resulting deformation field was applied to the distortion corrected EPI time-series, the mean EPI and the single-band reference EPI.	NA	NA	NA	To censor time-points significantly influences by noise the DVARS inference approach by Afyouni S. & Nichols T.E, (2017) was applied to each session for all subjects independently. Corrupted time-points were identified using the DVARSCalc function.	SPM12, 5mm smoothing with a fixed kernel in MNI152 space 	NA	4 sessions of 449 time points,  108 subjects 	NA	We applied an event-related design with each trial modeled as epochs of 4 sec duration with 3 parametric modulators [gain, loss, reaction time] orthogonalized by demeaning against the task and the respective preceding modulator. The canonical HRF were used for convolution including the temporal derivative. Additionally 6 motion regressors as obtained by realignment were added as regressors of no interest. Also time-points significantly influences by noise as flagged by the DVARS inference approach by Afyouni S. & Nichols T.E, (2017) were censored via an additional regressor. For each participant, all 4 sessions were modeled in one 1st Level design and contrast images for each regressor of interest were computed: [task, gain, loss, raction time].	pm	1	We applied one flexible factorial design to examine the effects of the following 4 factors of interest for the two groups, equal Indifference and equal Range: [task, gain, loss, reaction time] resulting in 8 conditions on the 2nd level. 	Mass Univariate	"1st-level: with autocorrelation model in SPM [AR(1) + w] and a high-pass filter of128 s
```

Comment:
Why did this change ?

Reply:
IMHO this file should just be ignored by codespell as these are the descriptions written by the team: I would treat that as "data" and not code.

Reply:
Yep, I agree on that. @yarikoptic is there a way to move this file away from the codespell reach ? And perhaps we should think of a more general rule.

Reply:
well, it has legit typos -- shouldn't we fix it?  indeed reviewing diff for it is tricky .  Here is the list of what changed in this file (if I have done it right):

```
--- a/narps_open/utils/description/analysis_pipelines_full_descriptions.tsv
+++ b/narps_open/utils/description/analysis_pipelines_full_descriptions.tsv
-peformed
+performed
-canoncial
+canonical
-behavoral
+behavioral
-indiviual
+individual
-nuissance
+nuisance
-nuissance
+nuisance
-overlayed
+overlaid
-implictly
+implicitly
-consited
+consisted
-testsing
+testing
-esclusion
+exclusion
-descrption
+description
-perfomed
+performed
-distorsion
+distortion
-infrormation.
+information.
-possitive
+positive
-smooting
+smoothing
-righ
+right
-(wihout loosing
+(without losing
-multplied
+multiplied
-eigth
+eighth
-derivitive
+derivative
-perfromed
+performed
-substraction
+subtraction
-avaliable
+available
-milimeters)
+millimeters)
-parameteres.
+parameters.
-paramters)
+parameters)
-poins
+points
-contast
+contrast
-exlcude
+exclude
-registeration
+registration
-potenial
+potential
-potenial
+potential
-transofrmation:
+transformation:
-tranformation
+transformation
-overlayed
+overlaid
-overal
+overall
-kernal
+kernel
-Gausian kernal
+Gaussian kernel
-equivalance
+equivalence
-thrid
+third
-thrid
+third
-dowloaded
+downloaded
-betwenn
+between
-knowlege.
+knowledge.
-cheking
+checking
-becasue
+because
-vizualization
+visualization
-vallue
+value
-vallue
+value
-reponse
+response
-reponses
+responses
-varaiables
+variables
-registeration
+registration
-kernal:
+kernel:
-esimated
+estimated
-interploation.
+interpolation.
-fo
+of
-signficant.
+significant.
-Euclidian
+Euclidean
-segementation
+segmentation
-structrual
+structural
-Euclidian
+Euclidean
-embeded
+embedded
-seperately.
+separately.
-threhold
+threshold
-perfomed.
+performed.
-embeded
+embedded
-embeded
+embedded
-embeded
+embedded
-Unkown
+Unknown
-Speparation
+Separation
-Currupted
+Corrupted
-co-ordinate
+coordinate
-co-ordinates:
+coordinates:
-co-ordinates:
+coordinates:
-signifcant
+significant
-signifcant
+significant
```

Reply:
Hi @yarikoptic, thanks for the changes to your PR. About this `analysis_pipelines_full_descriptions.tsv` file, I would prefer that we keep it as it is, and see it as raw data, as Remi mentioned. Even if I understand that this is a shame there are typos in it.

Reply:
you are the boss! I have added the file to ignore and force-pushed redone fixing with that file excluded.